Makefile

```diff
@@ -217,6 +236,59 @@ src.rpm source-rpm: spec dist
 	cp cri-resource-manager*.tar.bz2 ~/rpmbuild/SOURCES && \
 	rpmbuild -bs ~/rpmbuild/SPECS/cri-resource-manager.spec
 
+debian/%: packaging/deb.in/%
+	$(Q)echo "Generating debian packaging file $@..."; \
+	mkdir -p debian; \
+	eval `$(GIT_ID) .` && \
+	tarball=cri-resource-manager-$$gitversion.tar && \
+	cp $< $@ && \
+	sed -E -i "s/__PACKAGE__/cri-resource-manager/g" $@ && \
+	sed -E -i "s/__TARBALL__/$$tarball/g" $@ && \
+	sed -E -i "s/__VERSION__/$$debversion/g" $@ && \
+	sed -E -i "s/__AUTHOR__/$(USER_NAME)/g" $@ && \
+	sed -E -i "s/__EMAIL__/$(USER_EMAIL)/g" $@ && \
+	sed -E -i "s/__DATE__/$(BUILD_DATE)/g" $@ && \
+	sed -E -i "s/__BUILD_DIRS__/$(BUILD_DIRS)/g" $@
```

Comment:
You could combine multiple substitutions in one sed command with `-e` like
```
	sed -E -i -e "s/__PACKAGE__/cri-resource-manager/g" $@ && \
￼ 	          -e "s/__TARBALL__/$$tarball/g" $@ && \
...

```

Reply:
I'll fix that.